packages/o-spreadsheet-engine/src/types/chart/bubble_chart.ts

```diff
@@ -0,0 +1,27 @@
+import { ChartConfiguration } from "chart.js";
+import { Color } from "../..";
+import { AxesDesign, TitleDesign } from "./chart";
+import { LegendPosition, VerticalAxisPosition } from "./common_chart";
+import { ScatterChartRuntime } from "./scatter_chart";
+
+export type BubbleColorMode = "multiple" | Color;
+
+export interface BubbleChartDefinition {
+  readonly type: "bubble";
+  readonly xRange?: string;
+  readonly yRange?: string;
+  readonly labelRange?: string;
+  readonly title: TitleDesign;
+  readonly background?: Color;
+  readonly legendPosition: LegendPosition;
+  readonly axesDesign?: AxesDesign;
+  readonly showValues?: boolean;
+  readonly humanize?: boolean;
+  readonly sizeRange?: string;
+  readonly bubbleColor: BubbleColorMode;
+  readonly verticalAxisPosition: VerticalAxisPosition;
+}
+
+export type BubbleChartRuntime = Omit<ScatterChartRuntime, "chartJsConfig"> & {
+  chartJsConfig: ChartConfiguration<"bubble">;
+};
```

Comment:
should scatter & bubble chart runtime have a `masterChartConfig` in their runtime ? They're not zommable now right ? Or is the typing too annoying ?